build.sbt

```diff
@@ -57,6 +59,7 @@ lazy val scoverageSettings = {
     IntegrationTest / unmanagedSourceDirectories ++= Seq(baseDirectory.value / "it", baseDirectory.value / "test-common"),
     inConfig(IntegrationTest)(scalafixConfigSettings(IntegrationTest))
   )
+  .settings(headerSettings(IntegrationTest) ++ automateHeaderSettings(IntegrationTest))
```

Comment:
I didn't think you need this for it if you do line 53.  I thought this was required for accceptance tests etc.

Reply:
Seems you do as it didnt pick up the the files in /it otherwise.

Reply:
Maybe we should figure out why this is happening?